src/main/java/com/testfabrik/webmate/javasdk/jobs/JobRunState.java

```diff
@@ -0,0 +1,20 @@
+package com.testfabrik.webmate.javasdk.jobs;
+
+public enum JobRunState {
```

Comment:
Please add Javadoc describing that this is not the internal state representation of the JobRun but something sdk specific.